Source/BugsnagApiClient.h

```diff
@@ -7,7 +7,7 @@
 
 @class BugsnagConfiguration;
 
-typedef void (^RequestCompletion)(id data, BOOL success, NSError *error);
+typedef void (^RequestCompletion)(NSUInteger reportCount, BOOL success, NSError *error);
```

Comment:
`RequestCompletion` is an alias for `BSG_KSCrashReportFilterCompletion`, which did need to change. `RequestCompletion` only exists to avoid importing KSCrash internals headers into global scope.